packages/node_modules/@webex/plugin-meetings/src/meeting/in-meeting-actions.ts

```diff
@@ -13,36 +13,73 @@ export default class InMeetingActions {
   namespace = MEETINGS;
 
   canInviteNewParticipants: boolean = null;
-  canAdmitParticipant: boolean  = null;
+
+  canAdmitParticipant: boolean = null;
+
   canLock: boolean = null;
+
   canUnlock: boolean = null;
+
   canAssignHost: boolean = null;
+
   canStartRecording: boolean = null;
+
   canPauseRecording: boolean = null;
+
   canResumeRecording: boolean = null;
+
   canStopRecording: boolean = null;
+
   canRaiseHand: boolean = null;
+
   canLowerAllHands: boolean = null;
+
   canLowerSomeoneElsesHand: boolean = null;
 
-  get() {
-    return {
-      canInviteNewParticipants: this.canInviteNewParticipants,
-      canAdmitParticipant: this.canAdmitParticipant,
-      canLock: this.canLock,
-      canUnlock: this.canUnlock,
-      canAssignHost: this.canAssignHost,
-      canStartRecording: this.canStartRecording,
-      canPauseRecording: this.canPauseRecording,
-      canResumeRecording: this.canResumeRecording,
-      canStopRecording: this.canStopRecording,
-      canRaiseHand: this.canRaiseHand,
-      canLowerAllHands: this.canLowerAllHands,
-      canLowerSomeoneElsesHand: this.canLowerSomeoneElsesHand,
-    };
-  }
-
-  set(actions) {
+  canEnableClosedCaption: boolean = null;
+
+  canStartTranscribing: boolean = null;
+
+  canStopTranscribing: boolean = null;
+
+  isClosedCaptionActive: boolean = null;
+
+  isWebexAssistantActive: boolean = null;
+
+  canViewCaptionPanel : boolean = null;
+
+  isRealTimeTranslationEnabled: boolean = null;
+
+  canSelectSpokenLanguages : boolean = null;
+
+  /**
+   * Returns all meeting action options
+   * @returns {Object}
+   */
+  get = () :any => ({
```

Comment:
Typescript should infer the return type for you, please remove the `any`
If TS can not infer it then you should create a proper `interface` since you are know everything about the return type,